### PR TITLE
Updated allowed repository providers to include Git and GitHub in add…

### PR DESCRIPTION
**Pull request by balchen**
…ition to TfsGit

Bumped version number to 0.1.1.

### CODE REVIEW

**Comment by ppejovic** [comment]
Thank-you for useful contribution @balchen. Are you able to sign our CLA so I can merge the PR? It looks like the email address you used for your commits is not associated with your GitHub profile.

Fixes #10

**Comment by balchen** [comment]
Completed the paperwork now. Should be all set.

File: DedupeGitReposV0/DedupeGitRepos.ps1 [hunk 1]
```diff
@@ -6,14 +6,15 @@ $teamProject 		= $env:SYSTEM_TEAMPROJECT
 $workFolder 		= $env:AGENT_WORKFOLDER
 
 $buildDefinitionId 	= $env:SYSTEM_DEFINITIONID
-$repository 		= $env:BUILD_REPOSITORY_NAME
+$repository 		= $env:BUILD_REPOSITORY_ID
```
Comment (ppejovic) [inline DedupeGitReposV0/DedupeGitRepos.ps1]: @balchen in the case of GitHub the `Build.Repository.Name` predefined variable appears to return the string "repository" regardless of the actual repository name. This is used to key the shared repositories so needs to be unique. I've made an update to your PR to key off the `Build.Repository.ID` variable instead. It all looks good but I need to test further before merging.

Comment (balchen) [reply DedupeGitReposV0/DedupeGitRepos.ps1]: Good catch. I hadn't considered that.

**Pull request merged**